src/components/TradingViewChart/datafeed.tsx

```diff
@@ -420,6 +411,12 @@ export const useDatafeed = (currencies: Array<Currency | undefined>, pairAddress
                   c.low = c.close
                 }
               }
+              if (c.high > 1.1 * Math.max(c.open, c.close)) {
+                c.high = Math.max(c.open, c.close)
+              }
+              if (c.low < Math.min(c.open, c.close) / 1.1) {
+                c.low = Math.min(c.open, c.close)
+              }
```

Comment:
what is this? just want to understand

Reply:
@XiaoYhun provided this logic. It preventing data noise, price too high and too low